cwms-data-api/src/main/java/cwms/cda/data/dto/forecast/ForecastFile.java

```diff
@@ -0,0 +1,24 @@
+package cwms.cda.data.dto.forecast;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement(name = "alias")
+@XmlAccessorType(XmlAccessType.FIELD)
+public class ForecastFile {
+
+  @XmlAttribute

```

Comment:
Should include the unique identifiers for the forecast instant instead of the fcstInstCode.

blobCode should be replaced with the byte[] itself and the blob id.

Reply:
Actually I think this either goes away or becomes a generic representation the "binary_file_t" I'm working on for the text timeseries. But in that case it would be a member of the Instance.

Reply:
oh right, we revised this to be a single file rather than an unlimited set of files (zip 'em if needed)